tests/unit/cli/server/database/users/test_migrate.py

```diff
@@ -50,7 +50,7 @@ def test_migrate(monkeypatch, sync_db: "Session", cli_runner: CliRunner, cli: Ty
     assert user.role == UserRole.annotator
     assert user.api_key == "78a10b53-8db7-4ab5-9e9e-fbd4b7e76551"
     assert user.password_hash == "$2y$05$aqNyXcXRXddNj5toZwT0HugHqKZypvqlBAkZviAGGbsAC8oTj/P5K"
-    assert [ws.name for ws in user.workspaces] == ["tanya", "argilla", "team"]
+    assert set([ws.name for ws in user.workspaces]) == set(["tanya", "argilla", "team"])
```

Comment:
We are by default sorting user workspaces by `inserted_at` column so it's needs to be a problem with workspaces having the same `inserted_at`. We should check if we are using nanoseconds for `DateTime` columns.